### ISSUE TITLE
Mobile Navbar doesn't work

### ISSUE THREAD

**Issue by yung-megafone** [yung-megafone/NoRec-US#1]
Navbar on mobile devices is not responsive to touch. Desktop mode functions, and the Navbar menu functions ok on desktop per my last testing. 

**Need to dive into this ASAP**

**Comment by yung-megafone** [comment]
Fixed in latest release
